extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/AuthorityExtension.java

```diff
@@ -104,4 +122,24 @@ public ParticipantStore participantStore() {
     public CredentialsVerifier credentialsVerifier() {
         return new DummyCredentialsVerifier();
     }
+
+    @Provider(isDefault = true)
```

Comment:
shouldn't be a default provider. Default providers are intended for things like fallback services, such as an `in-memory` store. In this particular case there will be race conditions with the `resolverRegistry`, if that is also provided by a default provider.